conf/broker.conf

```diff
@@ -1031,6 +1031,8 @@ maxConcurrentHttpRequests=1024
 # For example: zk+hierarchical://localhost:2181/ledgers
 # The metadata service uri list can also be semicolon separated values like below:
 # zk+hierarchical://zk1:2181;zk2:2181;zk3:2181/ledgers
+# or
+# metadata-store:zk:localhost:2181  If we set -Dbookkeeper.metadata.bookie.drivers=org.apache.pulsar.metadata.bookkeeper.PulsarMetadataBookieDriver -Dbookkeeper.metadata.client.drivers=org.apache.pulsar.metadata.bookkeeper.PulsarMetadataClientDriver
```

Comment:
Since `bin/pulsar` will set these system properties unless `BK_METADATA_OPTIONS` is set to `none`, is this comment necessary? It might be better to update the complete documentation for `bookkeeperMetadataServiceUri` to match the default settings. 

https://github.com/apache/pulsar/blob/bbf56f651ebe357745b46654a1d9fb7dc4899fd7/bin/pulsar#L349-L355

Reply:
@gaozhangmin ping